float32 wrist_extension
float32 lift_position
---
bool success
